.github/workflows/release.yaml

```diff
@@ -14,9 +14,6 @@
 name: Release
 on:
   workflow_dispatch:
-  push:
-    branches:
-      - "main"
```

Comment:
we should not release on commit anymore.

Reply:
why? This will also create the PR for CLI integration.

This just affects VSC atm. Let's discuss tmrw.